src/components/Card/Card.module.scss

```diff
@@ -1,19 +1,30 @@
-.card {
+.card-background {
   background-color: var(--card-background-color, var(--color-brand-white-500));
-  border-radius: var(--modal-border-radius, var(--size-border-radius-md));
+}
+
+.card-radius {
+  border-radius: var(--card-border-radius, var(--size-border-radius-md));
+}
+
+.card-shadow {
   box-shadow: var(--card-box-shadow, var(--size-box-shadow-sm));
+}
 
-  &.subdued {
-    background-color: var(--card-subdued-background-color, var(--color-brand-grey-50));
-    box-shadow: none;
+.card-subdued {
+  background-color: var(--card-subdued-background-color, var(--color-brand-grey-50));
+  box-shadow: none;
+}
+
+.card-section-border {
+  + .card-section-border {
+    border-top: 1px solid var(--card-section-border-color, var(--color-brand-grey-100));
   }
 }
 
-.card-footer {
+.card-footer-background {
   background-color: var(--card-footer-background-color, var(--color-brand-grey-50));
-  border-color: var(--card-section-border-color, var(--color-brand-grey-100));
 }
 
-.card-section + .card-section {
```

Comment:
Why is this not necessary moving forward?

Reply:
it is still necessary, but written this way

```
.card-section-border {
  + .card-section-border {
    border-top: 1px solid var(--card-section-border-color, var(--color-brand-grey-100));
  }
}
```